### DISCUSSION TITLE
Please help me solve the permission issue

### DISCUSSION THREAD

**Discussion by noking1** [requarks/wiki#7141] [Error / Bug Report]
When I set 'PAGE RULES' for non-administrator user groups, once the 'path' is set with some rules, users in that user group cannot upload assets normally.
<img width="1165" alt="X20240425194305" src="https://github.com/requarks/wiki/assets/47624150/9b5a44ac-a08a-46a8-a8b2-4079fd6a94a1">
<img width="912" alt="X20240425194156" src="https://github.com/requarks/wiki/assets/47624150/4923d445-4ccf-4bc8-9768-f473ae4fc190">

**Comment by P4mes4n** [comment]
I think single and double letters are reserved for system pages and locales

**Reply by noking1** [reply]
Increasing the number of letters in the path to three or more, as you suggested, does not solve this issue.Thanks
